src/debugAdapter/goDebug.ts

```diff
@@ -321,6 +356,18 @@ class GoDebugSession extends DebugSession {
 		this.sendResponse(response);
 		log('ExceptionBreakPointsResponse');
 	}
+    
+    protected toDebuggerPath(path): string {
+        if (this.delve.remotePath.length === 0)
+            return this.convertClientPathToDebugger(path);
+        return path.replace(this.delve.program, this.delve.remotePath);
```

Comment:
I updated code so that if remotePath has trailing '/' or '\', it will remove it.  Assumption is that program will not have any trailing slashes.  Not sure if that is what you want.